toolkit/scripts/get_config_deps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# $1 - CONFIG_FILE
+# $2 - CONFIG_BASE_DIR
+
+if [[ $# -ne 2 ]]
+then
+    echo "ERROR: Must provide CONFIG_FILE and CONFIG_BASE_DIR to get_config_deps.sh" >&2
+    exit 1
+fi
+
+CONFIG_FILE="$1"
```

Comment:
Should also check that the file does in fact exist.
```suggestion
CONFIG_FILE="$1"
if [[ ! -f "$CONFIG_FILE" ]]
then
    echo "ERROR: Config file '$CONFIG_FILE' does not exist" >&2
    exit 1
fi
```